db/migrate/20160726061957_turn_default_value_to_lookup_value.rb

```diff
@@ -0,0 +1,19 @@
+class TurnDefaultValueToLookupValue < ActiveRecord::Migration
+  def up
+    LookupKey.unscoped.each do |key|
+      LookupValue.create(:match => 'default', :value => key.default_value, :lookup_key_id => key.id, :use_puppet_default => key.use_puppet_default)
+    end
+
+    remove_column :lookup_keys, :default_value
+    remove_column :lookup_keys, :use_puppet_default
+  end
+
+  def down
+    add_column :lookup_keys, :default_value, :string, :limit => 255
+    add_column :lookup_keys, :use_puppet_default, :boolean
+
+    LookupValue.where(:match => 'default').each do |lv|
+      LookupKey.find(lv.lookup_key_id).update_attributes(:default_value => lv.value, :use_puppet_default => lv.use_puppet_default)
+    end
+  end
```

Comment:
We need to delete the default records from the LookupValue table too.
And probably it would be better to wrap all those actions into a single transaction.